elasticdl/python/common/args.py

```diff
@@ -465,6 +465,14 @@ def parse_master_args(master_args=None):
             "checkpoint_filename_for_init is required for running "
             "prediction job"
         )
+    if not args.use_async and args.get_model_steps > 1:
+        raise ValueError(
+            "get_model_steps should be 1 when using synchronous SGD."
+        )
+    if args.use_async and args.grads_to_wait > 1:
```

Comment:
The default value for `grads_to_wait` is 2. Should we change it to 1 so async does not need to set it ?

Reply:
Good point. Done.